packages/cli/bin/merkur.mjs

```diff
@@ -138,4 +141,22 @@ program
     await custom({ args, commandArgs: cmd.args, command: COMMAND_NAME.CUSTOM });
   });
 
+// Load user-defined commands
+let userDefinedCommands = [];
+for (const { command, dir } of userDefinedCommandsPaths) {
+  const programCustom = new Command();
+  const commandModule = await import(path.join(dir, command));
+  const commandName = commandModule.default(({ program: programCustom })).name();
```

Comment:
I would add as argument new method https://github.com/mjancarik/merkur/blob/master/packages/cli/src/commandConfig.mjs which can be helpfull as example https://github.com/mjancarik/merkur/blob/master/packages/cli/src/commands/custom.mjs#L16.

Reply:
`createCommandConfig` method is new argument of `commandModule.default()`